src/main/java/org/opensearch/knn/index/codec/VectorAsArraySerializer.java

```diff
@@ -0,0 +1,41 @@
+/*
```

Comment:
nit: should we move these serializer classes under util package?

Reply:
"index.codec.util" sounds good!

Reply:
Great, moving classes to a new package, commit with corresponding changes is building